plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/BasicAuthenticationDetailView.java

```diff
@@ -101,6 +102,9 @@ public boolean isViewFor(Object object) {
 	public void onVisible(IObservableValue detailsViewModel, DataBindingContext dbc) {
 		bindWidgetsToInternalModel(dbc);
 		chkRememberToken.setText("&Save password (could trigger secure storage login)");
+		setCheckboxVisible(true);
+		chkRememberToken.getParent().update();
+		chkRememberToken.getParent().layout(true);
```

Comment:
looks like we have another GTK3 issue here? 
Can you please comment in code so that we know in maintenance why we need this?